diff --git a/node_modules/@fuel-ts/providers/dist/__generated__/operations.d.ts b/node_modules/@fuel-ts/providers/dist/__generated__/operations.d.ts
index f52511a..57b0b07 100644
--- a/node_modules/@fuel-ts/providers/dist/__generated__/operations.d.ts
+++ b/node_modules/@fuel-ts/providers/dist/__generated__/operations.d.ts
@@ -1,6 +1,6 @@
-/// <reference types="web" />
+// @ts-nocheck
 import type { GraphQLClient } from 'graphql-request';
-import type * as Dom from 'graphql-request/dist/types.dom';
+import type * as Dom from 'graphql-request/build/cjs';
 export type Maybe<T> = T | null;
 export type InputMaybe<T> = Maybe<T>;
 export type Exact<T extends {
